modules/accredited_representative_portal/app/controllers/accredited_representative_portal/application_controller.rb

```diff
@@ -4,15 +4,22 @@ module AccreditedRepresentativePortal
   class ApplicationController < SignIn::ApplicationController
     include SignIn::AudienceValidator
     include Authenticable
+    include Pundit::Authorization
```

Comment:
@ojbucao this is redundant with inherited `SignIn::ApplicationController`'s mixins